examples/cr-events.rs

```diff
@@ -93,9 +93,11 @@ fn main() {
                         new,
                         old: _,
                     } => (cr_type, new),
+                    _ => (CrType::Cr3, 0),
```

Comment:
Why do you match `_` to CR3 here ?
this means that anything that is ot Cr EventType will be considered a CR3 event ???